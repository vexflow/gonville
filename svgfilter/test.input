<glyph d="
converted: 1.23e+4
converted: 1.23e3
converted: 1.23e2
converted: 1.23e1
converted: 1.23e0
converted: 1.23e-1
converted: 1.23e-2
converted: 1.23e-3

converted: 0.00000000000123e+14
converted: 123000000e-6

converted: 000.000e+100
converted: 000.000e-100
converted: 0
" otherattr="
untouched: 1.23e2
">
untouched: 1.23e2
