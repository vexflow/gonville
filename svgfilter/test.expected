<glyph d="
converted: 12300
converted: 1230
converted: 123
converted: 12.3
converted: 1.23
converted: 0.123
converted: 0.0123
converted: 0.00123

converted: 123
converted: 123

converted: 0
converted: 0
converted: 0
" otherattr="
untouched: 1.23e2
">
untouched: 1.23e2
